build/automation/var/profile/dev.mk

```diff
@@ -104,3 +104,6 @@ TF_VAR_code_pipeline_branch_name := master
 TF_VAR_pipeline_topic_name := $(PROJECT_ID)-$(ENVIRONMENT)-pipeline-topic
 TF_VAR_pipeline_notification_name := $(PROJECT_ID)-$(ENVIRONMENT)-pipeline-notification
 TF_VAR_pipeline_chatbot_channel := $(PROJECT_ID)-cicd-slk-channel
+
+# Cloudwatch performance dashboard
+TF_VAR_cloudwatch_performance_dashboard_name := $(PROJECT_ID)-$(ENVIRONMENT)-performance
```

Comment:
rename to `TF_VAR_cloudwatch_monitoring_dashboard_name = $(PROJECT_ID)-$(ENVIRONMENT)-monitoring-dashboard` applies to other make files too